source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -11,7 +11,9 @@
 #include <playfab/PlayFabEventsDataModels.h>
 #include <playfab/PlayFabEventsInstanceApi.h>
 #include <playfab/PlayFabSettings.h>
+#if QOS_ENABLED
```

Comment:
I'm pretty tempted to move this entire test to a new file.
It's a bit out of place in "EventTest".
Then we can disable the whole file temporarily, and put a TODO: task number at the top.

Reply:
Done.